### PR TITLE
fix problematic bug introduced by new clldutils api for csv files due to quotechar

### PR DESCRIPTION
**Pull request by LinguList**
The character ```"``` is destroying the reading of orthoprofiles, since it is the quotechar as a default, so I added a keyword specifiying it should NOT assume any quotechar. This allows us to use the ```"``` as a character to be modified (just turned up when working on Uto-Aztekan)

### CODE REVIEW

**Comment by LinguList** [comment]
The alternative would be to write "" for all of those instances, but I think I'd prefer the quotechar=None solution, as it is less to explain to people...

File: segments/tokenizer.py [hunk 1]
```diff
@@ -170,9 +171,9 @@ class to build a trie data structure, which stores the possible Unicode
     In an orthography rules file, rules are given in order in regular
     expressions, e.g. this rule replaces a vowel followed by an <n>
     followed by <space> followed by a second vowel with first vowel
-    <space> <n> <space> second vowel, e.g.:
+    <space> <n> <space> second vowel, e.g.::
```
Comment (bambooforest) [inline segments/tokenizer.py]: @LinguList is the extra colon necessary (or a typo)?

**Comment by bambooforest** [comment]
I'm ok with the quotechar=None solution. Sounds good!

File: segments/tokenizer.py (hunk 1 above)
Comment (LinguList) [reply segments/tokenizer.py]: Rst! The markdown is generally corrupted in the docu. Ideally to be changed in all places. Maybe check with one of the numerous Python rst guides.

Comment (bambooforest) [reply segments/tokenizer.py]: Not corrupted if it was never my intention to use RST. But your point is noted. :)

**Pull request merged**